### PR TITLE
Add two new tools and extend another

### PR DESCRIPTION
**Pull request by micaeljtoliveira**
Add two new tools: one automatically fixes non-advective cells found in a mask file; the other combines two masks. Also modified the topog2mask.py utility to take treat cells with a fraction of ocean lower than a user-provided value as land.

### CODE REVIEW

**Comment by micaeljtoliveira** [comment]
This will be needed to fix https://github.com/COSIMA/access-om2/issues/265

**Comment by micaeljtoliveira** [comment]
@aekiss Would you mind reviewing this when you have some time? Thanks!

File: combine_masks.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    parser.add_argument('mask1', help='First mask to combine.')
+    parser.add_argument('mask2', help='Second mask to combine.')
+    parser.add_argument('grid', help='Ocean super-grid.')
+    parser.add_argument('output_mask', help='The combined mask.')
+    parser.add_argument('latitude', help='Take mask1 north of this value and mask2 otherwise.')
+
+    args = parser.parse_args()
+
+    if not os.path.exists(args.mask1) or not os.path.exists(args.mask2) or not os.path.exists(args.grid):
+        print('Error: one of {} {} was not found'.format(args.topog, args.input_mask, args.grid), file=sys.stderr)
```
Comment (aekiss) [inline combine_masks.py:53]: need another `{}`

File: combine_masks.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+import sys
+import os
+import numpy as np
+import argparse
+import netCDF4 as nc
+
+def fix_mask(mask1_filename, mask2_filename, grid_filename, lat, output_mask_filename):
+
+    lat = -60 # we will fix the mask south of this latitude
```
Comment (aekiss) [inline combine_masks.py:11]: remove line (set `lat=-60` as default in arg list?)

File: combine_masks.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    parser.add_argument('latitude', help='Take mask1 north of this value and mask2 otherwise.')
+
+    args = parser.parse_args()
+
+    if not os.path.exists(args.mask1) or not os.path.exists(args.mask2) or not os.path.exists(args.grid):
+        print('Error: one of {} {} was not found'.format(args.topog, args.input_mask, args.grid), file=sys.stderr)
+        parser.print_help()
+        return 1
+
+    fix_mask(args.mask1, args.mask2, args.grid, args.latitude, args.output_mask)
```
Comment (aekiss) [inline combine_masks.py:57]: should we exit with error if `fix_mask` returns 1?

**Review by aekiss — CHANGES_REQUESTED** [review]
thanks @micaeljtoliveira this looks great - just a few small comments

File: combine_masks.py (hunk 1 above)
Comment (micaeljtoliveira) [reply combine_masks.py]: Good catch! It's fixed now.

File: combine_masks.py (hunk 2 above)
Comment (micaeljtoliveira) [reply combine_masks.py]: Oh, that should not be there (it was a leftover of an older version of the script). The corresponding parser argument is mandatory, so no need to set a default value. I've fixed this, which in turn uncovered another issue, as the parser argument needs to be of type "float".

File: combine_masks.py (hunk 3 above)
Comment (micaeljtoliveira) [reply combine_masks.py]: Good idea. Done.

**Comment by micaeljtoliveira** [comment]
@aekiss Thanks for the careful review! I've addressed all your comments.

**Comment by aekiss** [comment]
thanks @micaeljtoliveira

**Pull request merged**